twigs/twigs.py

```diff
@@ -325,6 +326,35 @@ def main(args=None):
         parser_acr.add_argument('--no_code', action='store_true', help=argparse.SUPPRESS)
         parser_acr.add_argument('--sast', action='store_true', help=argparse.SUPPRESS)
 
+        
+        # Arguments required for Azure Container Registry container discovery 
+        parser_ecr = subparsers.add_parser ("ecr", help = "Discover AWS Container Registry (ECS) container images")
+        parser_ecr.add_argument('--registryId', help='The AWS Container Registry(AWS account ID) which needs to be inspected for all repositories.')
+        parser_ecr.add_argument('--repository', help='The fully qualified repository name which needs to be inspected. If tag is not given, latest will be determined and used.')
+        parser_ecr.add_argument ("--tag", help = "Specific tag for given repository")
+
+        parser_ecr.add_argument ("--repositoryType", help = argparse.SUPPRESS, choices=['public','private'], default='private')
```

Comment:
Do we need to remove the argparse.SUPPRESS here, since otherwise the repositoryType argument will not be visible to the user in help message and command-line?